client/src/cordova/plugin/apple/src/OutlinePlugin.swift

```diff
@@ -154,10 +154,12 @@ class OutlinePlugin: CDVPlugin {
     DDLogInfo("Invoking Method \(methodName) with input \(input)")
     Task {
       guard let result = OutlineInvokeMethod(methodName, input) else {
+        DDLogInfo("InvokeMethod \(methodName) got nil result")
```

Comment:
`Info` -> `Warn` since this is not the happy path? (but ignorable)

```suggestion
        DDLogWarn("InvokeMethod \(methodName) got nil result")
```